src/openrct2-unity/Assets/Scripts/Tools/SelectionTool.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using OpenRCT2.Unity;
+
+public class SelectionTool : MonoBehaviour
+{
+
+    [SerializeField] private CanvasManager canvasManager;
```

Comment:
'private' access modifier can be removed as it is the default.

Reply:
Even for SerializeFields?

Reply:
Yep. :) It's the default access modifier so it's implicitly there.